src/otx/core/utils/build.py

```diff
@@ -40,18 +40,43 @@ def build_mm_model(config: DictConfig, model_registry: Registry, load_from: str
     return model
 
 
-def get_classification_layers(config: DictConfig, model_registry: Registry, prefix: str = "") -> list[str]:
-    """Return classification layer names by comparing two different number of classes models."""
+def get_classification_layers(
+    config: DictConfig,
+    model_registry: Registry,
+    prefix: str = "",
+) -> dict[str, dict[str, int]]:
+    """Return classification layer names by comparing two different number of classes models.
+
+    Args:
+        config (DictConfig): Config for building model.
+        model_registry (Registry): Registry for building model.
+        prefix (str): Prefix of model param name.
+            Normally it is "model." since OTXModel set it's nn.Module model as self.model
+
```

Comment:
Since this is a util function in `otx.core` it would be great if there is an example section here showing the user/developer how to utilise this. It would also look good in the documentation?

or would it be too much effort now?

Reply:
alright, that's why I'm not the biggest mm fan :)